pennylane/optimize/nesterov_momentum.py

```diff
@@ -59,9 +59,8 @@ def compute_grad(self, objective_fn, x, grad_fn=None):
 
         shifted_x = unflatten(shifted_x_flat, x)
 
-        if grad_fn is not None:
-            g = grad_fn(shifted_x)  # just call the supplied grad function
-        else:
-            # default is autograd
-            g = autograd.grad(objective_fn)(shifted_x)  # pylint: disable=no-value-for-parameter
-        return g
+        g = get_gradient(objective_fn) if grad_fn is None else grad_fn
```

Comment:
Do we need to update the `compute_grad` docstring? The `Returns` part (could also include the suggestion in my previous comment if you adopt it there).